libs/ktem/flowsettings.py

```diff
@@ -87,7 +88,7 @@
     if len(KH_EMBEDDINGS) < 1:
         KH_EMBEDDINGS["openai"] = {
             "spec": {
-                "__type__": "kotaemon.embeddings.OpenAIEmbeddings",
```

Comment:
shouldn't this be `OpenAIEmbeddings` ?

Reply:
This should be `kotaemon.embeddings.OpenAIEmbeddings` for the the importlib module to be able to import it.